function.k

```diff
@@ -373,18 +459,27 @@ rule <k> pcsContractPart(C:Id, function F:Id (P1:FParameters) .FunQuantifiers
                <ctname> Cn:Id </ctname>
 	       ...
            </contractInstance> 
+
+  */   
```

Comment:
Is this rule commented-out? Do you still need it or can be removed?
From now on, I'd suggest to avoid leaving commented out code around (unless for some special need). 

Of course you can comment out code while you are working on something, but once you have fixed a problem or got an improved version of something, I'd suggest getting rid of the comments before doing commits. 

If something goes wrong and we want to come back to a previous version, we can always use the git history to retrieve that particular version of the code, so there's no need to keep many unused code around.